templates/base/storybook/.storybook/config.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { configure, addDecorator, addParameters } from '@storybook/react';
+import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withDesign } from 'storybook-addon-designs';
+import { withA11y } from '@storybook/addon-a11y';
+
+addDecorator(withKnobs);
+addDecorator(withA11y);
+addDecorator(withDesign);
+
+addDecorator((storyFn, context) => (
+  <React.Fragment>
+    {storyFn(context)}
```

Comment:
Ideally, this would be wrapped under ThemeProvider but since, we don't have themes for now. Is it okay to leave this as is @vinodloha ?

Reply:
Yeh fine for now